src/pynxtools/nomad/schema.py

```diff
@@ -469,17 +469,18 @@ class nexus definition.
     return section
 
 
-def __get_enumeration(xml_node: ET.Element) -> Optional[MEnum]:
+def __get_enumeration(xml_node: ET.Element) -> Tuple[Optional[MEnum], bool]:
     """
     Get the enumeration field from xml node
     """
     enumeration = xml_node.find("nx:enumeration", __XML_NAMESPACES)
     if enumeration is None:
-        return None
+        return None, None
```

Comment:
None is not bool why not false?

Reply:
None is neither True, nor False.

Reply:
clear your return type annotation for the second one is bool though i.e. constrained would using that be cleaner,, agree does not change your logic in L550

Reply:
done